etc/eth-contracts/.catalog-info.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: aurora-engine-token
+  description: |-
+    Aurora token deployed on the Aurora network
+  tags:
+    - contract
+    - aurora
+  links: []
+  annotations:
+    aurora.dev/security-tier: "1"
+spec:
+  owner: engine-team
+  type: contract
+  lifecycle: production
+  system: aurora-engine
+  deployedAt:
+    - contract:aurora/mainnet/0x8bec47865ade3b172a928df8f990bc7f2a3b9f79
```

Comment:
This lists only one particular contract for $AURORA token. Meanwhile, there are plenty of such tokens deployed. How should we deal with this case?

Reply:
It's a very good point and we will need to implement some discovery mechanism to ingest all deployed contracts automatically from the chain.

Let's keep this out of scope for now - a bit too complicated to solve in the short term.